File: programacao/Ex20.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char *argv[])
{
    float a, b, media;
     printf("Digite a primeira nota: ");
  scanf("%f", &a);
   printf("Digite a segunda nota: ");
  scanf("%f", &b);
   media = (a+(2*b))/3;
    printf("A media de %.1f e %.1f e igual a: %.2f\n", a, b, media);
    if (media>=5)
  {
      printf("aluno aprovado");
  }
   else
  {
      printf("aluno reprovado");
  }
  return 0;
}

File: python/Ex20.py
a = float(input("Digite a primeira nota: "))
b = float(input("Digite a segunda nota: "))
media = (a+(2*b))/3
print(f"o valor da media e: {media:.2f} ")
if(media >= 5):
    print(f"aluno aprovado")
else:
    print(f"aluno reprovado")